databases/business-sync/fragment-custom-reports.js

```diff
@@ -0,0 +1,49 @@
+{
+  channels: function(doc, oldDoc) {
+    var businessId = getBusinessId(doc, oldDoc);
+    return {
+      view: [ toSyncChannel(businessId, 'VIEW'), staffChannel ],
+      add: [ toSyncChannel(businessId, 'CHANGE_BUSINESS'), staffChannel ],
+      replace: [ toSyncChannel(businessId, 'CHANGE_BUSINESS'), staffChannel ],
+      remove: [ toSyncChannel(businessId, 'CHANGE_BUSINESS'), staffChannel ]
+    };
```

Comment:
I'm not 💯 on this. The privileges that we have defined in books for reports are different than for other objects. There is VIEW_REPORTS but no ADD or CHANGE. Because of this, I based all this on CHANGE_BUSINESS.